src/SignalR/common/Shared/PipeWriterStream.cs

```diff
@@ -6,6 +6,7 @@
 using System.IO;
 using System.Threading;
 using System.Threading.Tasks;
+using Microsoft.AspNetCore.Internal;
 
 namespace System.IO.Pipelines
 {
```

Comment:
Can this class be WriteOnlyPipeStream? https://github.com/aspnet/AspNetCore/blob/master/src/Http/Http/src/WriteOnlyPipeStream.cs

Reply:
You made it netcoreapp3.0 only, we need it for netstandard2.0